packages/server/src/AccountsServer.js

```diff
@@ -483,73 +482,97 @@ export class AccountsServer {
       throw new AccountsError('No such email address for user');
     }
     const token = generateRandomToken();
-    await this.db.addEmailVerificationToken(userId, address, token);
-
-    const siteUrl = this._options.siteUrl || config.siteUrl;
-    const verifyEmailUrl = `${siteUrl}/verify-email/${token}`;
-    await this.email.sendMail({
-      from: this.emailTemplates.verifyEmail.from ?
-        this.emailTemplates.verifyEmail.from : this.emailTemplates.from,
-      to: address,
-      subject: this.emailTemplates.verifyEmail.subject(user),
-      text: this.emailTemplates.verifyEmail.text(user, verifyEmailUrl),
-    });
+    await this.db.addEmailVerificationToken(user.id, address, token);
+
+    const resetPasswordMail = this._prepareMail(
+      address,
+      token,
+      user,
+      'verify-email',
+      this.emailTemplates.verifyEmail,
+      this.emailTemplates.from,
```

Comment:
`this.emailTemplates.verifyEmail.from ? this.emailTemplates.verifyEmail.from : this.emailTemplates.from`
This will allow user to overwrite from for each case if needed.